inc/term-sidebars.php

```diff
@@ -1,51 +1,65 @@
 <?php
+/**
+ * File for Largo's Term Sidebars
+ *
+ * @package Largo
+ */
 
 /**
  * Display the fields for selecting icons for terms in the "post-type" taxonomy
  */
 class Largo_Term_Sidebars {
 
-	function __construct() {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
 		add_action( 'edit_category_form_fields', array( $this, 'display_fields' ) );
 		add_action( 'edit_tag_form_fields', array( $this, 'display_fields' ) );
-		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts') );
-		add_action( 'edit_terms', array( $this, 'edit_terms' ) );
-		add_action( 'create_term', array( $this, 'edit_terms' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
+		add_action( 'edit_terms', array( $this, 'edit_terms' ), 10, 2 );
+		add_action( 'create_term', array( $this, 'edit_terms' ), 10, 2 );
 	}
 
 	/**
+	 * Return a list of taxonomies that term sidebars are enabled for
 	 *
+	 * @filter largo_get_sidebar_taxonomies
+	 * @return Array of taxonomy slugs
 	 */
-	function get_sidebar_taxonomies() {
-		if ( empty($this->_sidebar_taxonomies) ) {
+	public function get_sidebar_taxonomies() {
+		if ( empty( $this->_sidebar_taxonomies ) ) {
 			$this->_sidebar_taxonomies = apply_filters( 'largo_get_sidebar_taxonomies', array( 'category', 'post_tag', 'series' ) );
 		}
 		return $this->_sidebar_taxonomies;
 	}
 
 	/**
 	 * Renders the form fields on the term's edit page
+	 *
+	 * @param WP_Term $term The term for which the fields should be displayed.
+	 * @return nothing
 	 */
-	function display_fields( $term ) {
-		if ( !in_array( $term->taxonomy, $this->get_sidebar_taxonomies() ) ) {
-			// abort if the term doesn't belong to the taxonomies to have icons
+	public function display_fields( $term ) {
+		if ( ! in_array( $term->taxonomy, $this->get_sidebar_taxonomies(), true ) ) {
```

Comment:
This function should only be public if we can add to it function parameters that would allow overriding the `$term->taxonomy, $this->get_sidebar_taxonomies()` here.

Reply:
It makes no sense to override `$term->taxonomy`; you'd just change that in the `$term` passed. 

As for visibility, see the note on $this->get_sidebar_taxonomies() that my past comment talked about overriding.